core/eolearn/core/core_tasks.py

```diff
@@ -277,7 +277,7 @@ def __init__(
         self.shape: Union[None, Tuple[int, int, int], Tuple[int, int, int, int]]
 
         try:
-            self.shape_feature = self.parse_feature(shape)
+            self.shape_feature = self.parse_feature(shape)  # type: ignore[arg-type]
```

Comment:
Can you provide more detail here what exactly `mypy` is fussing about?

Reply:
`shape: Union[Tuple[int, ...], FeatureSpec]`, so it's not a suitable input for the feature parser. Which explains why the whole thing is in a `try except` statement.

Reply:
horrible, but resolving for now.